routes/bears.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const router = express.Router();
+const Bear = require('../models/bear');
```

Comment:
Please separate your code require statements from nodeJs require statements

Reply:
noted sir, would get on it